elements/modal.js

```diff
@@ -1,6 +1,122 @@
 const widget = require('cache-element/widget')
 const Modal = require('../lib/modal-element')
 const html = require('choo/html')
+var css = require('sheetify')
```

Comment:
should be a `const`. `const` is more explicit than `var` because it prevents the variable from ever being assigned to anything else.